images/base/Dockerfile

```diff
@@ -8,9 +8,10 @@ RUN \
   microdnf clean all && \
   rm -rf /var/cache/yum && \
   # The following setup is necessary so that this image can run as any user
-  mkdir -p /shared-home/.docker /shared-home/.ssh && chmod -R 0777 /shared-home && \
```

Comment:
This change is a problem. Needs to be discussed. We today build base images only from main but use them also for release-v0.16 builds. This would break them.